flake.nix

```diff
@@ -78,7 +85,7 @@
       }
 
       # Final target images
-      (import ./targets {inherit self nixos-generators microvm jetpack-nixos;})
+      (import ./targets {inherit self nixos-generators microvm jetpack-nixos netvm;})
```

Comment:
The function in targets/default.nix does not take netvm as an argument in the argument set

Reply:
Maybe some change is missing from this commit?